Packs/CommonScripts/ReleaseNotes/1_10_23.md

```diff
@@ -0,0 +1,15 @@
+
+#### Scripts
+##### URLReputation
+- Fixed an issue where using the script together with VirusTotal integration would return an error.
+- Updated the Docker image to: *demisto/python3:3.10.9.40422*.
+
+##### FileReputation
+- Fixed an issue where using the script together with VirusTotal integration would return an error.
+- Updated the Docker image to: *demisto/python3:3.10.9.40422*.
+
+##### DomainReputation
+- Fixed an issue where using the script together with VirusTotal integration would return an error.
```

Comment:
This issue is caused by [this feature](https://xsoar.pan.dev/docs/integrations/api_execution_metric_reporting), which happens to be used in VT.
 
```suggestion
- Fixed an issue where using the script together with integrations implementing API Execution Metric Reporting would return an error.
- Updated the Docker image to: *demisto/python3:3.10.9.40422*.

##### FileReputation
- Fixed an issue where using the script together with integrations implementing API Execution Metric Reporting would return an error.
- Updated the Docker image to: *demisto/python3:3.10.9.40422*.

##### DomainReputation
- Fixed an issue where using the script together with integrations implementing API Execution Metric Reporting would return an error.
```